blue_controllers:

  ### Controller for publishing to the joint_states topic

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 250

  ### Joint controllers

  torque_controller:
    type: effort_controllers/JointGroupEffortController
    joints:
      - base_roll_joint

  joint_position_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - base_roll_joint
    base_roll_joint:
      pid:
        p: 100
        i: 0
        d: 8
        i_clamp: 0
